app/js/components/shared/SortableTable.jsx

```diff
@@ -3,6 +3,7 @@ import React,  { Component } from "react";
 import PropTypes from "prop-types";
```

Comment:
I think this could be moved in react components in a new directory /src/components/table, what do you think?

Reply:
makes sense, would work on moving all the shared components from this module into react components.